internal/ignition/ignition.go

```diff
@@ -136,8 +155,25 @@ func (g *installerGenerator) generateLsoManifests(ctx context.Context, installer
 	return nil
 }
 
+func (g *installerGenerator) generateOcsManifests(ctx context.Context, installerPath string, envVars []string, ocsValidatorConfig ocs.OCSValidator) error {
```

Comment:
@nbalacha , there is already a function named `generateOCSManifests()` in `ignition.go` .